src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -146,6 +147,21 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 	}
 }
 
+export function validateCollectionName(collectionName: string): string | undefined | null {
+	// https://docs.mongodb.com/manual/reference/limits/#Restriction-on-Collection-Names
+	if (collectionName.length === 0) {
+		return "Collection name cannot be empty";
```

Comment:
Does this need to be case insensitive?

Reply:
I don't quite get you here.

Reply:
If you mean the system. prefix, the link mentioned only talks about the lower case s.

Reply:
Yes that's what I meant